tests/sla_tests.js

```diff
@@ -186,25 +243,52 @@ var getMockChannel = function() {
         }
       }, answeringDelay);
     };
+
     this.hangup = function(cb) {
       var self = this;
+      var channelFoundInBridges = false;
       this.wasHungup = true;
+
       setTimeout(function() {
         if (channels.length) {
+          bridgeChannels = bridgeChannels.filter(function(channel) {
+            if (channel === self.id) {
+              channelFoundInBridges = true;
+            }
+            return channel !== self.id;  
+          });
+
+          if (channelFoundInBridges) {
+            bridges[0].channels = bridgeChannels;
+            bridges[0].emit('ChannelLeftBridge', {channel: self,
+              bridge: bridges[0]}, {channel: self, bridge: bridges[0]});
+          }
+
           channels = channels.filter(function(channel) {
             return channel !== self;
           });
+
           self.emit('ChannelHangupRequest', {channel: self}, self);
           self.emit('ChannelDestroyed', {channel: self}, self);
           cb(null);
         }
       }, (asyncDelay/2));
     };
+
+    this.play = function(media, playbackObj) {
```

Comment:
What is the purpose of this.play? All it does is return null. It seems like an overkill way to assign null to some object.